internal/security/proxy/consumer_test.go

```diff
@@ -171,24 +171,24 @@ func TestCreateJWTToken(t *testing.T) {
 }
 
 /*func TestCreateOAuth2Token(t *testing.T) {
-	t.Skip()
-	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		w.WriteHeader(http.StatusOK)
-		w.Write([]byte(`{"token_type": "oauth2", "access_token": "test", "expires_in": 1442426001000}`))
-		if r.Method != "POST" {
-			t.Errorf("expected POST request, got %s instead", r.Method)
-		}
-
-		if r.URL.EscapedPath() != "/consumers/testuser/oauth2" {
-			t.Errorf("expected request to /consumers/testuser/oauth2, got %s instead", r.URL.EscapedPath())
-		}
-	}))
-	defer ts.Close()
-
-	co := Consumer{"testuser", &testConsumerRequestor{ts.URL}, &testConsumerConfig{ts.URL}}
-	_, err := co.createOAuth2Token()
-	if err != nil {
-		t.Errorf("failed to creat OAuth2 token for consumer")
-		t.Errorf(err.Error())
-	}
-}*/
+	 t.Skip()
+	 ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		 w.WriteHeader(http.StatusOK)
+		 w.Write([]byte(`{"token_type": "oauth2", "access_token": "test", "expires_in": 1442426001000}`))
+		 if r.Method != "POST" {
+			 t.Errorf("expected POST request, got %s instead", r.Method)
+		 }
+
+		 if r.URL.EscapedPath() != "/consumers/testuser/oauth2" {
+			 t.Errorf("expected request to /consumers/testuser/oauth2, got %s instead", r.URL.EscapedPath())
+		 }
+	 }))
+	 defer ts.Close()
+
+	 co := Consumer{"testuser", &testConsumerRequestor{ts.URL}, &testConsumerConfig{ts.URL}}
+	 _, err := co.createOAuth2Token()
+	 if err != nil {
+		 t.Errorf("failed to creat OAuth2 token for consumer")
+		 t.Errorf(err.Error())
+	 }
+ }*/
```

Comment:
Can we clean up this commented out dead code? You have that code available in the version-control repository anyway.

Reply:
will do.

Reply:
Are you planning on removing this code in this PR? I still see there's a large amount of commented out code being added here. I think keeping a git branch on your fork of the repository is a better place to keep this code rather than in the master branch.